caiman/source_extraction/cnmf/online_cnmf.py

```diff
@@ -389,11 +410,11 @@ def get_indices_of_pixels_on_ring(self, pixel):
     def fit_next(self, t, frame_in, num_iters_hals=3):
         """
         This method fits the next frame using the CaImAn online algorithm and
-        updates the object.
+        updates the object. Does NOT perform motion correction, see ``mc_next()``
```

Comment:
I think this is true

Reply:
Right, the logic of `fit_online()` is first `mc_next()` (line 1259), and then `fit_next()` (line 1271).